tests/custom_md_repo/conftest.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
I was wondering, should we move the `tests/custom_md_repo` tests to the existing `tests/repository` folder? They seem to fit "semantically" well there (as in, that's where I'd look for a test of a custom repo). 

We can probably add the `conftest.py`, the `dumymdrepo.py` fixture and the `test_custom_md_repo.py` there, at the top level folder.

Reply:
sorry,... I didn't this change because I don't know how to reuse conftest methods using a different settings with custom settings. Any idea how to reuse fixtures "mlserver" "rest_client" with a different settings?

Reply:
Good point! I think that the `prometheus_registry`, `settings` and `rest_client` fixtures have probably earned their place on the root `conftest.py`, as they're already used in a couple other test modules. That's completely out of the scope of this PR though - it may come in a future one though. 

So I think for now, we can just move both the fixtures and the tests jointly into a `tests/repository/test_custom_repository.py` file (don't worry - once I make the changes above, I'll remove the redundant ones so that it's only the `settings` fixture in there, which is the one that we're interested in modifying).

Reply:
Ok, I'll do it

Reply:
Done